src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs

```diff
@@ -183,9 +183,10 @@ private void GetExistingMemberInfo(INamedTypeSymbol containingType, out bool has
                         GetExistingMemberInfo(
                             info.ContainingType, out var hasEquals, out var hasGetHashCode);
 
+                        var typeDeclaration = info.SelectedDeclarations.First().Parent;
```

Comment:
i don't think this is right in VB.

Also, not sure what you get for selecting a field's variable declarator.  would it give the field or parent?

Better approach would be to use any of the helpers that give a containing type from a member.

Reply:
@CyrusNajmabadi I don't see such a helper from my guesses at Ctrl+T search and SyntaxGenerator intellisense. What do I look for?

Reply:
I'll look.  But first let me know if you still need this.  You may not anymore after your most recent discussion with @sharwell it looks like.  LMK.

Reply:
I will still need it because I'm still going to have to know whether I should call `.WithNullability(NullableAnnotation.Annotated)` or not when generating the Equals method.

Reply:
Found .ConstructWithNullability, see update.